pkg/autogen/rule.go

```diff
@@ -326,3 +344,19 @@ func updateRestrictedFields(pbyte []byte, kind string) (obj []byte) {
 	obj = []byte(strings.ReplaceAll(string(obj), "metadata", "spec.template.metadata"))
 	return obj
 }
+
+func updateCELFields(pbyte []byte, kind string) (obj []byte) {
```

Comment:
Can we re-use `updateGenRuleByte()`?

Reply:
Actually CEL variables are object, oldObject, request, params and authorizer.  We can have policy as follows:
```
  rules:
    - name: test-rule
      match:
        any:
        - resources:
            kinds:
              -  '*'
      celPreconditions:
           - name: 'rbac'
             expression: 'request.resource.group != "rbac.authorization.k8s.io"'
      validate:
        cel:
          expressions:
           - expression: "!object.metadata.name.contains('demo') || object.metadata.namespace == 'demo'"
 ```

Therefore, I believe we should call `updateGenRuleBytes()` as well as `updateCELFields()`.